src/llv8.cc

```diff
@@ -1577,6 +1748,16 @@ std::string JSObject::InspectDescriptors(Map map, Error& err) {
       continue;
     }
 
+    if (!descriptors.IsFieldDetails(details)) {
```

Comment:
Please remove this (lines 1751 to 1760). This was removed by https://github.com/nodejs/llnode/pull/221 to unbreak our tests and probably was reinserted here during rebase.

Honestly, I'm surprised tests are passing even though this code is still here.